library/src/layout/page.rs

```diff
@@ -97,6 +97,8 @@ pub struct PageElem {
     ///   - `right`: The right margin.
     ///   - `bottom`: The bottom margin.
     ///   - `left`: The left margin.
+    ///   - `inside`: The inner margin for two-sided documents.
+    ///   - `outside`: The outer margin for two-sided documents.
```

Comment:
Should probably elaborate a bit here to mention that they are exclusive with `left` and `right`.

Reply:
Done.